packages/agent_sdk/memory/sqlite_store.py

```
@@ -6,17 +1,161 @@
+            params.append(since.isoformat())
+        
+        where_clause = " AND ".join(conditions) if conditions else "1=1"
+        params.append(limit)
+        
+        query = f"""
+            SELECT id, agent_id, task_id, context, result, timestamp 
+            FROM agent_interactions 
+            WHERE {where_clause} 
+            ORDER BY timestamp DESC 
+            LIMIT ?
+        """
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)